packages/rich-list-core/__tests__/controller/AddressParser/dftx/accountToUtxos.test.ts

```diff
@@ -0,0 +1,51 @@
+import { JsonRpcClient } from '@defichain/jellyfish-api-jsonrpc'
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { RawTransaction } from '@defichain/jellyfish-api-core/src/category/rawtx'
+import { AddressParser } from '../../../../src/controller/AddressParser'
+
+describe('AccountToUtxosParser', () => {
+  const container = new MasterNodeRegTestContainer()
+  let apiClient!: JsonRpcClient
+
+  let sender!: string
+  let rec1!: string
+  let rec2!: string
+  let rec3!: string
+  let rawTx!: RawTransaction
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForWalletCoinbaseMaturity()
+    apiClient = new JsonRpcClient(await container.getCachedRpcUrl())
+
+    sender = await container.getNewAddress()
+    rec1 = await container.getNewAddress()
+    rec2 = await container.getNewAddress()
+    rec3 = await container.getNewAddress()
```

Comment:
https://github.com/DeFiCh/jellyfish/blob/a76864a9dbf966e688827d9df22d48e531b7ce2a/packages/jellyfish-testing/src/testing.ts#L33

Awhile back this feature was introduced in `jellyfish-testing` to provide the ability to use named address for testing.